ci/azure-pipelines.yml

```diff
@@ -4,9 +4,9 @@
 
 name: $(SourceBranchName)-$(Date:yyyyMMdd)$(Rev:.rrr)
 trigger:
-- master
+- main
 pr:
-- master
+- main
```

Comment:
I would suggest pulling this out into a spearate pr, but we're probably in a broken state until it gets done.

Add a Comment in the commit message.

Need a follow up issue, for revising the documentation where the default branch name shows up.

Reply:
looks like #36 does most of that